observability/akka-http-metrics/src/main/scala/com/daml/metrics/http/DamlHttpMetrics.scala

```diff
@@ -14,11 +14,19 @@ class DamlHttpMetrics(metricsFactory: Factory, component: String) extends HttpMe
     Map(Labels.ServiceLabel -> component)
   )
 
-  override val requestsTotal: Meter = metricsFactory.meter(httpMetricsPrefix :+ "requests")
-  override val latency: Timer = metricsFactory.timer(httpMetricsPrefix :+ "requests")
+  override val requestsTotal: Meter =
+    metricsFactory.meter(httpMetricsPrefix :+ "requests", "Total number of HTTP requests received")
+  override val latency: Timer =
+    metricsFactory.timer(httpMetricsPrefix :+ "requests", "The duration of the HTTP requests")
   override val requestsPayloadBytes: Histogram =
-    metricsFactory.histogram(httpMetricsPrefix :+ "requests" :+ "payload" :+ Histogram.Bytes)
+    metricsFactory.histogram(
+      httpMetricsPrefix :+ "requests" :+ "payload" :+ Histogram.Bytes,
+      "The size of the payloads received in the HTTP requests",
```

Comment:
Taking inspiration from the [Node Exporter](https://github.com/prometheus/node_exporter) metrics, we might want to describe histograms as `distribution`:
```suggestion
      "Distribution of the sizes of payloads received in HTTP requests.",
```